shared/src/languages.ts

```diff
@@ -104,6 +104,10 @@ function getModeFromExtension(ext: string): string | undefined {
         case 'h':
             return 'cpp'
 
+        // CUDA
+        case 'cu':
```

Comment:
Add cuda header files too: 

```suggestion
        case 'cu':
        case 'cuh':
```